lighthouse-core/audits/dobetterweb/link-blocking-first-paint.js

```diff
@@ -55,31 +56,43 @@ class LinkBlockingFirstPaintAudit extends Audit {
       };
     }
 
-    let totalSpendTime = 0;
-    const results = artifact.items
-      .filter(item => item.tag.tagName === tagFilter)
-      .map(item => {
-        totalSpendTime += item.spendTime;
+    const filtered = artifact.filter(item => item.tag.tagName === tagFilter);
 
-        return {
-          url: item.tag.url,
-          label: `delayed first paint by ${item.spendTime}ms`
-        };
-      });
+    let startTime = Number.MAX_VALUE;
+    let endTime = 0;
+    startTime = filtered.reduce((t, item) => Math.min(t, item.startTime), startTime);
 
+    const results = filtered.map(item => {
+      endTime = Math.max(item.endTime, endTime);
+
+      return {
+        url: URL.getDisplayName(item.tag.url),
+        totalKb: `${Math.round(item.transferSize / 1024)} KB`,
```

Comment:
should we include the unit alongside the table value (what you have) or just in the header?